me.amankhanna.opendeck.yaml

```diff
@@ -0,0 +1,35 @@
+id: me.amankhanna.opendeck
+
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+
+command: opendeck
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/ninjadev64/OpenDeck/releases/download/v2.1.2/opendeck_2.1.2_amd64.deb
```

Comment:
why not building from source?

Reply:
https://github.com/tauri-apps/tauri/issues/11210#issuecomment-2408877526

Reply:
this is not a justification.

Reply:
I see. There seems to be no dependency manifest generator for Deno, so how generate a dependency manifest for Deno?

Reply:
I see that the absence of a pnpm dependency manifest generator has been enough reason to allow other applications to build from the .deb (see GitButler for an example). Could the same please be done in this case as there is no Deno dependency manifest generator?